### ISSUE TITLE
Parent role is not defined for menu items

### ISSUE THREAD

**Issue by claudiahub** [backstage/backstage#11163]
## Expected Behavior

Users on screen readers are aware of the context on menu(s).
In particular, the parent role as menu, menubar or group should be defined for the element tags above where role = 'menu-items' is defined. 

## Actual Behavior

1. Navigate to Docs menu item from the sidebar.
2. Navigate to the left menu component and press F12 to open developer tool.
3. Verify whether the menu-items under `Personal` and `My company` have a parent role defined for its above tags. 

## Context

User Impact: If parent role is not defined then it later becomes hard for screen reader users to understand the context.

A11y Conformance Level: A

WCAG Success Criteria: 4.1.2

## Environment

Browser Information: 
Google Chrome v100.0.4896.127
Firefox 99.0.1
Safari 15.4
